lib/utils/debounce.html

```diff
@@ -68,7 +68,19 @@
     }
   /**
    * Creates a debouncer if no debouncer is passed as a parameter
-   * or it cancels an active debouncer otherwise.
+   * or it cancels an active debouncer otherwise. For example, the following
+   * code can be called multiple times within a microtask and the provided
+   * function will be debounced (called once at the next microtask checkpoint):
+   *
+   * let job = Polymer.Debouncer.debounce(job, Polymer.Async.microTask, () => {
+   *   console.log('debounced!');
+   * });
```

Comment:
Isn't it important for the value of `job` to be preserved across multiple calls here? If I copy this code into an event handler, will it debounce correctly?

That is, wouldn't you usually store the job like `this._job = Polymer.Debouncer.debounce(_job ...)`